src/control/cmd/daos/object.go

```diff
@@ -159,7 +159,7 @@ func (cmd *objQueryCmd) Execute(_ []string) error {
 	for i, shard := range layout.Shards {
 		cmd.Infof("grp: %d", i)
 		for j, replica := range shard.Replicas {
-			cmd.Infof("replica %d %d", j, replica.Rank)
+			cmd.Infof("replica %d %d:%d", j, replica.Rank, replica.Target)
```

Comment:
don't think this change is related to this PR, is it?

Reply:
Sort of...My reasoning for this PR was to spit out the layout and without the target, it's not really complete.